src/integration/pages/mediaAssetPage/pidgin/__snapshots__/amp.test.js.snap

```diff
@@ -266,14 +266,34 @@ exports[`AMP Media Asset Page I can see a bulleted list item 1`] = `"<Media Asse
 
 exports[`AMP Media Asset Page I can see a bulleted list item with link: <Media Asset Page> 1`] = `"http://www.bbc.com/afrique/23248423"`;
 
+exports[`AMP Media Asset Page I can see related content: 'I no know say I different for society until pipo begin look me one kain' 1`] = `"/pidgin/media-53580248"`;
```

Comment:
Just an observation - it's a shame it doesn't mention anything about mostWatched in the snapshot, but perhaps something we could look at improving later - perhaps having separate data-e2es for related content + most watched.

Reply:
Sure. we can improve that later